src/app/query/bitcoin/contract/check-dlc-link-attestor-health.ts

```diff
@@ -0,0 +1,17 @@
+export async function checkDlcLinkAttestorHealth(attestorUrl: string): Promise<boolean> {
+  let response: Response;
+
+  try {
+    response = await fetch(`${attestorUrl}/health`, {
+      method: 'get',
+    });
+  } catch (error) {
+    return false;
+  }
+
+  if (!response.ok) {
+    return false;
+  }
+
+  return true;
+}
```

Comment:
Is it necessary to hoist the response var above and declare in the try/catch?

Isn't this the same as? 

```suggestion
  try {
    const response = await fetch(`${attestorUrl}/health`, {
      method: 'get',
    });
    
    return response.ok;
    
  } catch (error) {
    return false;
  }
}
```

Reply:
You are right, I removed the unnecessary lines!